packages/flutter/lib/src/rendering/debug.dart

```diff
@@ -54,6 +58,82 @@ List<String> debugDescribeTransform(Matrix4 transform) {
   return matrix;
 }
 
+bool _extensionsInitialized = false;
+
+void initServiceExtensions() {
+  if (_extensionsInitialized)
+    return;
+
+  _extensionsInitialized = true;
+
+  assert(() {
+    registerExtension('flutter', _flutter);
+    // TODO: Expose debugDumpApp().
```

Comment:
We usually use the format `TODO(username)`. Though honestly it might be best just to file bugs for this kind of thing.